src/app/4-data-binding/data-binding/data-binding.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component } from '@angular/core';
+import { NgForm } from '@angular/forms'; 
+
+@Component({
+  selector: 'app-data-binding',
+  templateUrl: './data-binding.component.html',
+  styleUrl: './data-binding.component.scss'
+})
+
+export class DataBindingComponent {
+  oneWayText: string = "";
+  twoWaysText: string = "";
+
+clickMe() {
```

Comment:
No estas identando correctamente, favor de hacerlo correctamente o en su defecto instalar como te había dicho la extensión de prettier en tu vscode y darle formato a tu proyecto.

Reply:
fixed, prettier seems to be working now :)